python/erdos/message.py

```diff
@@ -66,10 +68,10 @@ class WatermarkMessage(Message):
         timestamp: The timestamp for which this is a watermark.
     """
 
-    def __init__(self, timestamp: Timestamp):
+    def __init__(self, timestamp: Timestamp) -> None:
         super(WatermarkMessage, self).__init__(timestamp, None)
 
-    def __str__(self):
+    def __str__(self) :
```

Comment:
I'm not sure why the CI is not running. But, could you please run the following from your `python` directory?

```bash
black .
isort --profile black .
flake8 --inline-quotes=double ./
```
and commit any changes that these tools make?

Reply:
We need to manually approve and run the CI because Hiva is a first-time committer.

Reply:
Thanks! @HivaMohammadzadeh1 the CI should now show the errors in formatting, please take a look and address the issues.

Reply:
Thank you! Will do.

Reply:
After running `flake8 --inline-quotes=double ./`
I get this error. Is this normal?
```
usage: flake8 [options] file file ...
flake8: error: unrecognized arguments: --inline-quotes=double
```
It does not show me any errors in formatting, so I think that error might not be normal.

Reply:
Perfect that was the problem. I ran and resolved the issues and committed.